api-server/justfile

```diff
@@ -0,0 +1,74 @@
+# load environment vars from .env
+set dotenv-load := true
+
+# display this help
+default:
+  just --list
+
+# build the api-server docker image
+build:
+  docker-compose build api
```

Comment:
Is there way to have this tagged with a default and pass through optional other tags. That way all local builds have a tag vs release builds